sympy/core/tests/test_basic.py

```diff
@@ -255,7 +255,7 @@ def test_atomic():
     assert _atomic(g(x + h(x))) == {g(x + h(x))}
     assert _atomic(g(x + h(x)), recursive=True) == {h(x), x, g(x + h(x))}
     assert _atomic(1) == set()
-    assert _atomic(Basic(1,2)) == {Basic(1, 2)}
+    assert _atomic(Basic(S(1), S(2))) == set()
```

Comment:
Why has this changed?

Reply:
It seems that `_atomic` takes into account the case of `Basic` with non-`Basic` arguments and considers them atoms.

Reply:
Actually, it's weird that it returns nothing, this seems more like a bug that is somehow exposed now